homeassistant/components/hue/strings.json

```diff
@@ -47,5 +53,15 @@
       "remote_double_button_long_press": "Both \"{subtype}\" released after long press",
       "remote_double_button_short_press": "Both \"{subtype}\" released"
     }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "allow_how_groups": "Allow Hue groups",
```

Comment:
@frenck this key looks wrong. Not sure if I should start a new issue for this...

Reply:
Thanks! Will address that 👍